src/Pickles/Pickles/PicklesModule.cs

```diff
@@ -145,18 +144,10 @@ protected override void Load(ContainerBuilder builder)
             builder.RegisterType<StrikeMarkdownProvider>();
             builder.RegisterType<MarkdownProvider>();
 
-            builder.Register<IMarkdownProvider>(
-                c =>
-                    {
-                        if (FeatureSwitcher.Feature<AlternateMarkdownProvider>.Is().Enabled)
-                        {
-                            return c.Resolve<StrikeMarkdownProvider>();
-                        }
-                        else
-                        {
-                            return c.Resolve<MarkdownProvider>();
-                        }
-                    }).SingleInstance();
+            builder.Register(
```

Comment:
A request for the future: try not to give in to resharper's temptation to optimize code that is unrelated to the feature you're working on. I know it's difficult :-)

I won't reject the PR because of this but please keep it in mind for the future.